Blockzilla/Info.plist

```diff
@@ -2,8 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>SentryDSN</key>
```

Comment:
I know this was all Xcode. But could you also revert this change to keep the file history clean?